arbitrum-docs/useful-addresses.mdx

```diff
@@ -31,59 +29,87 @@ Here's some contract addresses that may be useful and/or of interest to those ex
+| L2 Arb-Custom Gateway | <AEL address="0x096760F208390250649E3e8763348E783AEF5562" chainID={42161} /> | <AEL address="0xbf544970E6BD77b21C6492C281AB60d0770451F4" chainID={42170} /> | <AEL address="0x8b6990830cF135318f75182487A4D7698549C717" chainID={421613} /> |
+| L1 Weth Gateway       | <AEL address="0xd92023E9d9911199a6711321D1277285e6d4e2db" chainID={1} />     | <AEL address="0xE4E2121b479017955Be0b175305B35f312330BaE" chainID={1} />     | <AEL address="0x6e244cD02BBB8a6dbd7F626f05B2ef82151Ab502" chainID={5} />      |
+| L2 Weth Gateway       | <AEL address="0x6c411aD3E74De3E7Bd422b94A27770f5B86C623B" chainID={42161} /> | <AEL address="0x7626841cB6113412F9c88D3ADC720C9FAC88D9eD" chainID={42170} /> | <AEL address="0xf9F2e89c8347BD96742Cc07095dee490e64301d6" chainID={421613} /> |
+| L1 Weth               | <AEL address="0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2" chainID={1} />     | <AEL address="0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2" chainID={1} />     | <AEL address="0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6" chainID={5} />      |
+| L2 Weth               | <AEL address="0x82aF49447D8a07e3bd95BD0d56f35241523fBab1" chainID={42161} /> | <AEL address="0x722E8BdD2ce80A4422E880164f2079488e115365" chainID={42170} /> | <AEL address="0xe39Ab88f8A4777030A534146A9Ca3B52bd5D43A3" chainID={421613} /> |
 
 #### Third party gateways
 
 These are the addresses of some other gateways that have been deployed and integrated with the token bridge
 
-|                     | Mainnet: Arbitrum One                                                                                                 |
-| ------------------- | --------------------------------------------------------------------------------------------------------------------- |
-| L1 Dai Gateway      | [0xD3B5b60020504bc3489D6949d545893982BA3011](https://etherscan.io/address/0xD3B5b60020504bc3489D6949d545893982BA3011) |
-| L2 Dai Gateway      | [0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65](https://arbiscan.io/address/0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65)  |
-| L1 Livepeer Gateway | [0x6142f1C8bBF02E6A6bd074E8d564c9A5420a0676](https://etherscan.io/address/0x6142f1C8bBF02E6A6bd074E8d564c9A5420a0676) |
-| L2 Livepeer Gateway | [0x6d2457a4ad276000a615295f7a80f79e48ccd318](https://arbiscan.io/address/0x6D2457a4ad276000A615295f7A80F79E48CcD318)  |
+|                     | Mainnet: Arbitrum One                                                        |
+| ------------------- | ---------------------------------------------------------------------------- |
+| L1 Dai Gateway      | <AEL address="0xD3B5b60020504bc3489D6949d545893982BA3011" chainID={1} />     |
+| L2 Dai Gateway      | <AEL address="0x467194771dAe2967Aef3ECbEDD3Bf9a310C76C65" chainID={42161} /> |
+| L1 Livepeer Gateway | <AEL address="0x6142f1C8bBF02E6A6bd074E8d564c9A5420a0676" chainID={1} />     |
+| L2 Livepeer Gateway | <AEL address="0x6d2457a4ad276000a615295f7a80f79e48ccd318" chainID={42161} /> |
```

Comment:
prefer to preserve original letter case
`0x6D2457a4ad276000A615295f7A80F79E48CcD318`